src/interp.cc

```diff
@@ -2903,6 +2909,26 @@ Result Thread::Run(int num_instructions) {
       case Opcode::F64X2Sqrt:
         CHECK_TRAP(SimdUnop<v128, int64_t>(FloatSqrt<double>));
         break;
+
+      case Opcode::F32X4ConvertSI32X4:
+        CHECK_TRAP(SimdUnop<v128, int32_t>(SimdConvert<float, int32_t>));
+        break;
+
+      case Opcode::F32X4ConvertUI32X4:
+        CHECK_TRAP(SimdUnop<v128, uint32_t>(SimdConvert<float, uint32_t>));
+        break;
+
+      case Opcode::F64X2ConvertSI64X2:
+        CHECK_TRAP(SimdUnop<v128, int64_t>(SimdConvert<double, int64_t>));
+        break;
+
+      case Opcode::F64X2ConvertUI64X2:
+        #if COMPILER_IS_MSVC && defined(_M_X64)
```

Comment:
Hmm, I'd prefer to keep the #if in `config.h.in`. I forgot that `wabt_convert_*` is not a function otherwise. Can you instead convert the macro in `config.h.in` to an inline function and always use that here? Sorry for the extra effort!

Reply:
Sure, I will do that.

Reply:
Hm, looking at the appveyor build, it looks like it breaks on Windows. Not quite sure why though.